Makefile

```diff
@@ -1,5 +1,5 @@
 PROJECT = mysql_poolboy
 DEPS = mysql poolboy
-dep_mysql = git https://github.com/mysql-otp/mysql-otp.git 0.8.0
+dep_mysql = git https://github.com/mysql-otp/mysql-otp.git 1.0.0
 dep_poolboy = git https://github.com/devinus/poolboy.git 1.4.2
```

Comment:
It would be nice if poolboy could also be the same version as in the rebar file. It's not nice that these can differ at all. I will we could have a way to unit test that the rebar and erlang.mk dependencies are identical...